src/upgrades/pre_0_9_0/pubd_objects_migration.rs

```diff
@@ -331,9 +326,6 @@ struct OldRepository {
 
     rrdp: OldRrdpServer,
     rsync: RsyncdStore,
-
-    #[serde(default = "RepoStats::default")]
-    stats: RepoStats,
```

Comment:
Why is the serialization of `OldRepository` being changed?

Reply:
It was just removed, so it will be ignored, because it no longer needs to be included in the new structure for the migration.